{"status":"implemeted","returnType":"void","arguments":[{"status":"implemeted","description":"Message that describes exception event.","name":"message","type":"string","required":false},{"status":"implemeted","description":"- A custom type\n- Application\nDo not enter another predefined type; types are not generated by CFML applications. If you specify Application, you need not specify a type for cfcatch.","name":"type","type":"string","required":false},{"status":"implemeted","description":"addional detailed description of the exception.","name":"detail","type":"string","required":false},{"status":"implemeted","description":" A custom error code that you supply.","name":"errorcode","type":"string","required":false},{"status":"implemeted","description":"extended information to the exception.","name":"extendedInfo","type":"string","required":false},{"status":"implemeted","description":"Throws a Java exception from a CFML tag. This attribute is mutually exclusive with all other argments of this function.","name":"object","type":"any","required":false}],"description":"Throws a developer-specified exception, which can be caught with a cfcatch tag","source":"C:\\Users\\mark.drew\\Desktop\\railos\\railo-express-4.1.0.011-nojre\\lib\\ext\\railo-server\\context\\library\\function\\throw.cfm","argMax":6,"name":"throw","argMin":0,"type":"cfml","argumentType":"fixed"}